.changeset/itchy-apples-dream.md

```diff
@@ -0,0 +1,10 @@
+---
+'polaris-for-vscode': patch
+'@shopify/polaris-icons': patch
+'@shopify/polaris': patch
+'@shopify/polaris-tokens': patch
+'polaris.shopify.com': patch
+'@shopify/stylelint-polaris': patch
```

Comment:
@chazdean when running changesets please only tag the `polaris.shopify.com` repo.
```suggestion
'polaris.shopify.com': patch
```

Reply:
Got it!